modules/publication/README.md

```diff
@@ -0,0 +1,48 @@
+# Publication
+
+## Purpose
+The publication module allows users to propose research projects based on LORIS 
```

Comment:
cc @mohadesz @cmadjar  per Monday meeting - 

"Publication" strongly suggests this module is for tracking **Papers** underway/assigned/published.  

However says it's for **Project Proposals** -- i.e. researchers request to perform a series of Analyses, via access to specific datasets (like genomics). 

* This extra sense of "project" confuses our documentation -> suggest "Analysis Proposal" or "Research Proposal"? 

Typically, one research analysis project can generate multiple publications.  And/or one publication and result from several analyses.